src/CLI/CLI.php

```diff
@@ -142,6 +153,72 @@ public function task(string $name): Task
         return $task;
     }
 
+    /**
+     * If a resource has been created return it, otherwise create it and then return it
+     *
+     * @param string $name
+     * @param bool $fresh
+     * @return mixed
+     * @throws Exception
+     */
+    public function getResource(string $name, bool $fresh = false): mixed
+    {
+        if ($name === 'utopia') {
```

Comment:
I think this was designed against a memory leak. Is it relevant in this context? Should we prevent the CLI instance as a nested dependency itself?

Reply:
Not sure, not fully aware what might be. May be we can leave as is and later change if any problem arises?

Reply:
I'll ping @TorstenDittmann about it.

Reply:
removed this for now